framework/db/src/main/java/com/cloud/utils/db/Filter.java

```diff
@@ -69,7 +75,7 @@ public Filter(Filter that) {
         that._limit = null;
     }
 
-    public void addOrderBy(Class<?> clazz, String field, boolean ascending) {
+    public void addOrderBy(Class<?> clazz, String field, boolean ascending, String tableAlias) {
```

Comment:
```suggestion
    public void addOrderBy(Class<?> clazz, String field, boolean ascending) {
        return addOrderBy(clazz, field, ascending, null)
    }

    public void addOrderBy(Class<?> clazz, String field, boolean ascending, String tableAlias) {
```

Reply:
good call @vishesh92 , makes this change a lot smaller and the rest of the code cleaner (less <null>)

Reply:
Great, thanks @vishesh92 - fixed